src/components/DownloadCard/index.tsx

```diff
@@ -180,6 +188,15 @@ export function DownloadCard(props: Props) {
           onFormSubmit(e);
         }}
       >
+        <label for="">Data type</label>
```

Comment:
This is displaying above the other dropdowns it should be inline with them side by side